flix-core/src/test/java/com/epam/mentoring/flixcore/service/MovieServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.mentoring.flixcore.service;
+
+import com.epam.mentoring.flixcore.FlixCoreApplication;
+import com.epam.mentoring.flixcore.model.Genre;
+import com.epam.mentoring.flixcore.model.Movie;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = FlixCoreApplication.class)
+public class MovieServiceTest {
+
+    @Autowired
+    private MovieService movieService;
+
+    private Movie lordOfTheRings;
+    private Movie titanic;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
its always good to name method correctly, i know that in most cases its setUp but it tells nothing. 
It would be better to call it setUpInitialMovies(), but check my next comment.
Same with tearDown() method.

Reply:
Renamed